Dependencies/xr/Include/XR.h

```diff
@@ -209,7 +210,10 @@ namespace xr
                     };
 
                     std::array<float, DEFAULT_CONTROLLER_AXES> Axes;
-                    std::array<Button, DEFAULT_CONTROLLER_BUTTONS> Buttons;
+                    std::array<Button, DEFAULT_CONTROLLER_BUTTONS + SPECIAL_CONTROLLER_BUTTONS> Buttons;
```

Comment:
does this really need to be a static size or could we use a vector with the correct sizes and avoid having an additional count value?